### PR TITLE
gateio fetchMyTrades true

### PR DESCRIPTION
**Pull request by ndubel**
fetchMyTrades is works for gateio

### CODE REVIEW

File: js/gateio.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ module.exports = class gateio extends Exchange {
                 'fetchOrderTrades': true,
                 'fetchOrders': true,
                 'fetchOrder': true,
+                'fetchMyTrades': true,
```
Comment (kroitor) [inline js/gateio.js]: We can't really set it to `true` without adding an implementation for it (see how it is implemented in other exchanges). Would you want to give it try?

Comment (frosty00) [reply js/gateio.js]: I wrote fetchMyTrades a while back, must have forgot the metadata :)

Comment (kroitor) [reply js/gateio.js]: @frosty00 lol, i'm a blind idiot then ) Thx for pointing that out! )

@ndubel pardon for the confusion!

Comment (kroitor) [reply js/gateio.js]: @frosty00 we've sent you a message to your email address (the address we got from you earlier), check it out when you have time for it) or let us know if you haven't received anything.

**Pull request merged**